src/pages/components/radio-button/style.mdx

```diff
@@ -10,21 +10,21 @@ tabs: ['Usage', 'Style', 'Code', 'Accessibility']
 
 | Class                                   | Property         | Color token |
 | --------------------------------------- | ---------------- | ----------- |
-| `.bx--label`                            | text color       | `$text-02`  |
-| `.bx--radio-button__label`              | text color       | `$text-01`  |
-| `.bx--radio-button__appearance`         | border           | `$ui-05`    |
-| `.bx--radio-button__appearance`         | background-color | `$ui-01`    |
-| `.bx--radio-button__appearance:checked` | border           | `$ui-05`    |
-| `.bx--radio-button__appearance::before` | dot              | `$ui-05`    |
+| `.bx--label`                            | text color       | `$text-secondary`  |
+| `.bx--radio-button__label`              | text color       | `$text-primary`  |
+| `.bx--radio-button__appearance`         | border           | `$icon-primary`    |
+| `.bx--radio-button__appearance`         | background-color | `$background`    |
+| `.bx--radio-button__appearance:checked` | border           | `$border-inverse`    |
```

Comment:
We treat the radio and checkboxes like icons so this would also just be `$icon-primary`